docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature: `1303` User can now purge cached ethereum transactions and exchange data. After purging, the data will be queried from their respective sources. This might take some time depending on the amount of entries that will be queried.
```

Comment:
Please specify what data means for exhanges also: (deposits/withdrawals/trades).

> After purging, the data will be queried from their respective sources

This is not correct or at least can be misunderstood.

How about :

>  The next time data is fetched, the respective sources will be queried to re-populate the local DB cache.

And then you can append the:

> This might take some time depending on the amount of entries that will be queried.